src/d3a/models/area/__init__.py

```diff
@@ -143,6 +145,7 @@ def __init__(self, name: str = None, children: List["Area"] = None,
             if external_connection_available and self.strategy is None else None
         self.should_update_child_strategies = False
         self.external_connection_available = external_connection_available
+        self.state = MarketClearingState()
```

Comment:
Can you please explain why you added this to the area and not re-added it to the market class?
If you want to keep it here (which i am not a fan of), please rename to something like (self.market_clearing_state) in order to not get confused with the state that is used for restarting the CN. Thanks!

Reply:
> Can you please explain why you added this to the area and not re-added it to the market class?

Added to market class: https://github.com/gridsingularity/d3a/pull/1195/commits/9fd4444ed7113cb99129fc29b1cd803d298dd580